azure-pipelines.yaml

```diff
@@ -42,28 +42,14 @@ jobs:
     inputs:
       version: $(DotNetCoreVersion)
 
-  - task: NuGetToolInstaller@0
-    displayName: 'Install NuGet'
-    inputs:
-      versionSpec: 5.x
-
-  - task: NuGetCommand@2
-    displayName: 'NuGet Restore'
-    inputs:
-      restoreSolution: Xamarin.Android.Tools.sln
-      feedsToUse: config
-      nugetConfigPath: NuGet.config
-
-  - task: MSBuild@1
+  - script: dotnet build Xamarin.Android.Tools.sln -bl:$(Build.ArtifactStagingDirectory)/build.binlog
```

Comment:
It may be preferable to use DotNetCoreCLI@2 here, though it shouldn't make much of a difference

```yaml
  - task: DotNetCoreCLI@2
    displayName: 'Build solution Xamarin.Android.Tools.sln'
    inputs:
      projects: Xamarin.Android.Tools.sln
      arguments: -bl:$(Build.ArtifactStagingDirectory)/build.binlog
```

Reply:
I thought I would also have to put `command: build` or `command: pack`.

Does it *do* anything extra that is helpful? Like insert warnings inline in PRs? Something else?

Otherwise a one-line `script:` block seems simpler?

Reply:
The `command: build` param is used by default, so it doesn't need to be set when building. You would need to set `command: pack` for packing though.

I think the only advantage to using the task over a script is that it will probe multiple locations for `dotnet`, but since we're using the `UseDotNet` task there shouldn't be a difference either way.